presto-docs/src/main/sphinx/optimizer/cost-in-explain.rst

```diff
@@ -2,14 +2,14 @@
 Cost in EXPLAIN
```

Comment:
yeah. I agree that it's good to have documentation about how costs work in addition to information about basic stats.